test/functional/test_framework/authproxy.py

```diff
@@ -102,6 +96,8 @@ def _request(self, method, path, postdata):
                    'User-Agent': USER_AGENT,
                    'Authorization': self.__auth_header,
                    'Content-type': 'application/json'}
+        if os.name == 'nt': # Windows somehow does not like to re-use connections
```

Comment:
I guess this is meant as a temporary work-around until we find out *why*? (might want to add a comment in that regard)